---
layout: page
title: Software
---

Writing scientific software is a big component of my [research]({{url}}/research.md).

### [arbutus](https://github.com/mwpennell/arbutus) 

Assess the adequacy of phylogenetic models of quantitative trait evolution. 

### [geiger](https://github.com/mwpennell/geiger-v2)

A broad suite of tools for fitting macroevolutionary models to phylogenetic comparative data. 

### [chromer](https://github.com/ropensci/chromer)

Programmatic access to the the (awesome) [Chromosome Counts Database](http://ccdb.tau.ac.il/services/). Developed in collaboration with the [ROpenSci](https://ropensci.org) team.

### [phyndr](https://github.com/richfitz/phyndr)

Use taxonomic or topological information to maximize the overlap of phylogenetic and comparative data.

### [taxonlookup](https://github.com/wcornwell/taxonlookup)

A dynamic taxonomic resource released as a R data package. Currently this only covers plants but we hope to eventually expand it to other groups.

### [capwire](https://github.com/mwpennell/capwire)

Software for estimating population census size from non-invasive genetic sampling. 
